module/VuFind/tests/fixtures/alma/holdings-without-mappings.json

```diff
@@ -145,18 +145,20 @@
         {
             "id": "1111",
             "source": "Solr",
-            "callnumber": "HB Ab",
+            "callnumber": "",
```

Comment:
Is there a strong reason to remove the call number from the test fixture? Seems like adding the subfield back to minimize diffs and exercise more code might be worthwhile... but if this was done for a reason, I obviously don't object to the change. :-)

Reply:
It's not relevant for electronic availability. The test returned physical item availability previously because the fixture contained wrong field (AVA instead of AVE).